.github/ISSUE_TEMPLATE/Standard.md

```diff
@@ -34,10 +34,11 @@ Where is this issue occurring?
 **Version Number:**
 **Reproducible in staging?:**
 **Reproducible in production?:**
+**Email of affected user:**
```

Comment:
NAB; Think we should say Email / phone here in case their primary login is a phone number?